variables.scad

```diff
@@ -14,23 +14,26 @@ SensorHole_ledge = 1.0;
 
 wall_thickness = 2;
 
-UsbCover_height = 10;
+UsbCover_height = 9.7;
 UsbCover_foot_height = 1.5;
 UsbCover_width = 15.5;
 UsbCover_depth = 6.5;
 UsbCover_foot_length = 2.5;
 UsbCover_radius = 1.5;
-UsbCover_magnet_size = 3.0; // increase if your printer needs more clearance for press fit
-UsbCover_magnet_depth = 3.3;
+
+UsbCover_r1 = 1.8; // the rounded corners with smaller radius
+UsbCover_r2 = 2; // the rounded corners with bigger radius
+UsbCover_magnet_size = 3.0; // decrease if magnets are too loose
+UsbCover_magnet_clearance = 0.25; // increase if your printer needs more clearance for press fit
```

Comment:
I don't understand why the clearance and the size are modelled differently. Why not just increase the magnet size if your hole is too narrow on your printer?

Reply:
The trick is that it is much easier to have a narrow opening but a wide hole for the magnets: you just have to squeeze the magnets through the narrow hole at the top after which they comfortably sit in their roomy hole with plenty clearance. This way the opening can be much tighter and will grip the magnet. More detailed discussion [here](https://github.com/openbikesensor/OpenBikeSensor3dPrintableCase/issues/14)